docs/changelog.rst

```diff
@@ -0,0 +1,94 @@
+Changelog
+~~~~~~~~~
+
+0.24.0 (September 2022)
+-----------------------
+
+* Updated pytket version requirement to 1.7.
```

Comment:
The change in `setup.py` is to 1.6; are we going to wait for 1.7 and update there?

Reply:
Yes